Kickstarter-iOS/Views/Cells/CommentCell.swift

```diff
@@ -0,0 +1,22 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal final class CommentCell: UITableViewCell, ValueCell {
+  internal override func bindStyles() {
+    super.bindStyles()
+
+    let cellBackgroundColor = UIColor.ksr_white
```

Comment:
I think we don't need this constant since we are only using it to set the backgroundColor on line 14